ts/components/conversation/MemberList.tsx

```diff
@@ -80,7 +78,8 @@ class MemberItem extends React.Component<MemberItemProps> {
         onClick={this.handleClick}
       >
         {this.renderAvatar()}
-        <span className="name-part">{nameAndPubKey}</span>
+        <span className="name-part">{name}</span>
+        <span className="pubkey-part">{pubkey}</span>
```

Comment:
If I remember correctly, I removed that one two weeks ago. Why do you need it there?

Reply:
Mentions needs to be split into two parts in order to render correctly as two span elements.